### ISSUE TITLE
Accessibility:Step Input: The corresponding label “Quantity” is not read, only the placeholder text ‘Things quantity’ is read.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3491]
Description: The corresponding label “Quantity” is not read, only the placeholder text ‘Things quantity’ is read.

Expected: Quantity label should be read by the screen reader.

Screenshot:

![image](https://user-images.githubusercontent.com/32538291/94650045-9096dc80-0313-11eb-9fd2-75e9298d6038.png)

**Comment by Betrozov** [comment]
This issue related to `FormFieldComponent`

**Comment by puru-hk** [comment]
Tested it is working as expected.
